etcdserver/util.go

```diff
@@ -52,3 +52,34 @@ func numConnectedSince(transport rafthttp.Transporter, since time.Time, self typ
 	}
 	return connectedNum
 }
+
+// longestConnected chooses the member with longest active-since-time
+// as a leader transferee, since it indicates the stability of the member.
```

Comment:
we should not mention leader transferee. there might be other use cases in the future.